SU-SWS/ace-sdssgryphon
# SDSS-499: removing min height on sm/xm

# READY FOR REVIEW

# Summary
- Removed the min-height from sm/xs view

# Review By (Date)
- ASAP

# Urgency
- Higth

# Steps to Test

1. Checkout this branch
2. Look at a banner in a sm/xs size and verify there is no extra gap.
3. Verify the banner is set the correct height in other window sizes

# Affected Projects or Products
- SDSS Subtheme

# Associated Issues and/or People
[- SDSS-499](https://stanfordits.atlassian.net/browse/SDSS-499)

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
